.github/workflows/build-and-test.yml

```diff
@@ -35,19 +35,6 @@ jobs:
           distribution: 'temurin'
           cache: 'gradle'
 
-      - name: Git secrets setup
-        run: |
-          git clone https://github.com/awslabs/git-secrets.git ~/git-secrets
-          cd ~/git-secrets
-          git checkout 99d01d58ebcc06e237c0e3f3ff5ae628aeef6aa6
-          sudo make install
-
-      - name: Secrets check
-        run: |
-          sudo ln -s "$(which echo)" /usr/local/bin/say
-          ./minnie-kenny.sh --force
-          git secrets --scan-history
-
```

Comment:
We no longer need minnie-kenny, since InfoSec has centralized secret detection. Using minnie-kenny imposes setup burden on local environments.

One could argue it's worth it even though we also have centralized detection. Reviewers, what do you think?